app/dev_mode/views.py

```diff
@@ -9,6 +10,15 @@
 
 logger = logging.getLogger(__name__)
 
+logging.getLogger('flask_cors').level = logging.DEBUG
+
+
+@dev_mode_blueprint.route('/dev/render-template', methods=['POST'])
+@cross_origin()
```

Comment:
This could do with looking at by somebody who knows what they are doing (ie. not me), I just hacked this in to get it to work.